html/main.js

```diff
@@ -0,0 +1,183 @@
+const cardColors = [
+    "red",
+    "orange",
+    "yellow",
+    "olive",
+    "green",
+    "teal",
+    "blue",
+    "violet",
+    "purple",
+    "pink",
+    "brown",
+    "grey",
+    "black"
+];
+
+const messageKindCommand = "command";
+const messageOperationWrite = "write";
+const messageOperationDelete = "delete";
+
+let cardsParent = $("#cards-parent");
+let modalUi = $(".ui.modal.add-command-modal");
+let modalForm = $(".ui.modal.add-command-modal #form-modal");
+
+$(".ui.card").hover(
+    function () {
+        $(this).addClass("raised");
+    }, function () {
+        $(this).removeClass("raised");
+    }
+);
+
+$(".ui.transparent.fluid.input").focusin(
+    function () {
+        $(this).children("div").show(100);
+    }
+).focusout(
+    function () {
+        $(this).children("div").hide(100);
+    }
+);
+
+$(".sortable").sortable({
+    placeholder: "card"
+});
+
+$(".add-command").click(function () {
+    $(".ui.modal.add-command-modal").modal("show");
+});
+
+cardsParent.on("click", ".delete", function () {
+    sendDeleteCommand($(this).parents(".ui.card").find(".header input").val());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['sendDeleteCommand' was used before it was defined.](https://app.codacy.com/manual/ttimt/Short_Cutteer/pullRequest?prid=4845233)